### PR TITLE
Fix getDevices permissions when no kind is supplied

### PR DESCRIPTION
**Pull request by Talljoe**
The check for active devices for the Safari workaround skips getting permission if no kind is supplied.

### CODE REVIEW

File: src/room/DeviceManager.ts [hunk 1]
```diff
@@ -85,4 +84,10 @@ export default class DeviceManager {
 
     return device?.deviceId;
   }
+
+  private hasActiveDevice(kind?: MediaDeviceKind): boolean {
```
Comment (lukasIO) [inline src/room/DeviceManager.ts:88]: Thanks for the PR! I think this makes sense, only the naming on this one might be confusing. The check in this function does not really say anything about an `activeDevice`, but only whether or not a promise is in the map.

Comment (Talljoe) [reply src/room/DeviceManager.ts:88]: Hrmm. My understanding is the map contains devices that have been requested and the fact that they are "in use" in some fashion makes Safari unhappy. Is the objection that the devices might have been requested but not active? I'm trying to capture a name that explains why a device in the map is significant (for Safari).

Comment (Talljoe) [reply src/room/DeviceManager.ts:88]: That makes sense. I'll come up with a new name.

**Review by lukasIO — APPROVED** [review]
Thanks, looks good to me now!

**Pull request merged**